src/main/java/com/healthsync/ui/LoginScene.java

```diff
@@ -63,10 +63,12 @@ private static VBox createContent(Stage stage) {
                 String role = authenticatedUser.getRole();
                 switch (role) {
                     case "Patient" -> {
-                        PatientScene patientScene = new PatientScene();
+                        PatientScene patientScene = new PatientScene(authenticatedUser, stage);
+                        stage.setMinWidth(1157);
+                        stage.setMinWidth(1160);
                         stage.setScene(patientScene);
+                        stage.centerOnScreen();
                         stage.setTitle("HealthSync - Patient");
```

Comment:
Is there a reason these stage components (setMinWidth, centerOnScreen, etc.) can't be moved inside patientScene? Also is the 1157, 1160 thing a duplicate

Reply:
The second line is a duplicate, 1157 should be the right size.

Those other lines should be able to go in PatientScene, I think I just put them there because they're similar to setScene and setTitle.